vpr/src/pack/cluster.cpp

```diff
@@ -2532,28 +2727,83 @@ static void add_cluster_molecule_candidates_by_attraction_group(t_pb* cur_pb,
                                                                 t_cluster_placement_stats* cluster_placement_stats_ptr,
```

Comment:
above routine: attraction groups can be very large, we only add some of the attraction group molecules for efficiency reasons

if you call it multiple times you will get different answers